core/btdiscovery.cpp

```diff
@@ -229,6 +229,11 @@ void BTDiscovery::getBluetoothDevices()
 			result.name = dev.callObjectMethod("getName", "()Ljava/lang/String;").toString();
 			qDebug() << "paired Device type" << btType << "with address" << result.address;
 			btPairedDevices.append(result);
+			if (btType == 3) { // DEVICE_TYPE_DUAL
```

Comment:
No, for a btType == 3, 2 paired devices are written. The normal BT one (without LE:) and a second one with LE: . We want to be able to choose.